framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1554,6 +1554,14 @@ protected void applyInferredAnnotations(AnnotatedTypeMirror type, Value as) {
         applier.applyInferredType(type, as.getAnnotations(), as.getUnderlyingType());
     }
 
+    /**
+     * To add annotations to the type of method or constructor parameters, add a {@link
```

Comment:
I would add `{@intheritDoc}` because there is good information in the superclass that is lost in this class.